dockstore-webservice/src/main/resources/openapi.yaml

```diff
@@ -4841,6 +4841,83 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Entry"
+  "/workflows/path/service/{repository}":
+    post:
+      tags:
+        - workflows
+      summary: Create a service for the given repository (ex. dockstore/dockstore-ui2).
```

Comment:
I am generally not a fan of having path parameters with "/" in it.  Leads to confusion when I make a repository like "dockstore/upsertVersion"